internal/server/audit/webhook/client.go

```diff
@@ -40,7 +42,9 @@ func (w *webhookClient) signPayload(payload []byte) []byte {
 }
 
 // NewHTTPClient is the constructor for a HTTPClient.
-func NewWebhookClient(logger *zap.Logger, url, signingSecret string, httpClient *retryablehttp.Client) Client {
+func NewWebhookClient(logger *zap.Logger, url, signingSecret string, maxBackoffDuration time.Duration) Client {
+	httpClient := retryablehttp.NewClient()
+	httpClient.Logger = template.NewLeveledLogger(logger)
```

Comment:
Do we need to be setting the RetryWay to maxBackoffDuration here?
```suggestion
	httpClient.Logger = template.NewLeveledLogger(logger)
	httpClient.RetryWait = maxBackoffDuration
```

Reply:
@GeorgeMac great catch!